Rules/UseConsistentIndentation.cs

```diff
@@ -134,6 +134,8 @@ public override IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string file
             var onNewLine = true;
             var pipelineAsts = ast.FindAll(testAst => testAst is PipelineAst && (testAst as PipelineAst).PipelineElements.Count > 1, true).ToList();
             int minimumPipelineAstIndex = 0;
+            var lParenSkippedIndentation = new Stack<bool>();
```

Comment:
Given the complexity of our algorithm here has increased to the point where we're maintaining a stack, it might be worth trying to capture the general procedure in a comment

Reply:
added comment in b3bf36d